assets/src/dashboard/components/dropdown/stories/index.js

```diff
@@ -93,24 +93,47 @@ export const _transparent = () => {
   );
 };
 
+const categoryDemoData = [
+  {
+    label: <span>{'All Categories'}</span>,
+    value: 'all',
+    selected: true,
+  },
+  { label: 'Arts and Crafts', value: 'arts_crafts' },
+  { label: 'Beauty', value: 'beauty' },
+  { label: 'Cooking', value: 'cooking' },
+  { label: 'News', value: 'news' },
+  { label: 'Sports', value: 'sports' },
+  { label: 'News', value: 'news' },
+  { label: 'UNCLICKABLE', value: false },
+];
+
 export const _panel = () => {
+  const [statefulDemoData, setStatefulDemoData] = useState(categoryDemoData);
+
+  const updateDemoDataState = (datumToUpdate) => {
+    const newDemoData = statefulDemoData.map((item) => {
+      if (item.value === datumToUpdate) {
+        return { ...item, selected: !item.selected };
+      }
+      return item;
+    });
```

Comment:
this isn't perfect, just demoing the hook up of `onChange`

Reply:
We can wrap this in a `useCallback`

Reply:
sure!